src/web/nextui/next.config.js

```diff
@@ -1,13 +1,29 @@
-const outputType = process.env.NEXT_PUBLIC_PROMPTFOO_BUILD_STANDALONE_SERVER
-  ? 'standalone'
-  : 'export';
+const API_PORT = process.env.API_PORT || '15500';
+
+if (process.env.NODE_ENV === 'development') {
+  process.env.NEXT_PUBLIC_PROMPTFOO_REMOTE_API_BASE_URL =
+    process.env.PROMPTFOO_REMOTE_API_BASE_URL || `http://localhost:${API_PORT}`;
+  process.env.NEXT_PUBLIC_PROMPTFOO_SHARE_API_URL = `http://localhost:${API_PORT}`;
```

Comment:
if we're in development we're always going to be pointing to our local API. Locally this runs on a different port than the web app